### PR TITLE
Feat/mongo db ds

### PR DESCRIPTION
**Pull request by hainv-ohio**

### CODE REVIEW

**Comment by brianlagunas** [comment]
# DO NOT MERGE

Since we have not been able to create a working visualization with this connector, it's not ready for merging. I will review and see if I can figure out why it's not working.

**Comment by brianlagunas** [comment]
This still needs to be rebased with the main branch

**Comment by brianlagunas** [comment]
this still has merge conflicts

**Review by brianlagunas — CHANGES_REQUESTED** [review]
fix merge conflcits

**Review by brianlagunas — CHANGES_REQUESTED** [review]

File: src/Reveal.Sdk.Dom/Data/DataSources/MongoDBDataSource.cs [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 namespace Reveal.Sdk.Dom.Data
 {
-    internal class MongoDBDataSource : DatabaseDataSource
+    public class MongoDBDataSource : DataSource
```
[the remark this answers is not in the document]
Comment (hainv-ohio) [reply src/Reveal.Sdk.Dom/Data/DataSources/MongoDBDataSource.cs]: It was a mistake when I tried to make it work, I misunderstood some things, one of them I undertood `Database` field wasn't needed. Reverted

**Review by brianlagunas — COMMENTED** [review]
fix the merge conflicts

**Comment by brianlagunas** [comment]
never mind, I'll do it

**Pull request merged**